### PR TITLE
Migrate consumer configuration for kafka-authorization

### PR DESCRIPTION
**Pull request by Diewa**
I migrated the kafka.authorization configuration and additionally created a mock configuration yml to have a visualization of how it will look like after the migration. Maybe this will allow us to organize the configurations better.

### CODE REVIEW

File: hermes-consumers/src/test/java/pl/allegro/tech/hermes/consumers/consumer/rate/maxrate/HierarchicalCacheMaxRateRegistryTest.java [hunk 1]
```diff
@@ -32,10 +32,10 @@ public class HierarchicalCacheMaxRateRegistryTest extends ZookeeperBaseTest {
     private final MaxRatePathSerializer pathSerializer = new MaxRatePathSerializer();
     private final SubscriptionsCache subscriptionsCache = mock(SubscriptionsCache.class);
     private final ConfigFactory configFactory = new MutableConfigFactory();
-    private final String cluster = configFactory.getStringProperty(Configs.KAFKA_CLUSTER_NAME);
+    private final String cluster = "primary-dc";
```
Comment (Kornelos) [inline hermes-consumers/src/test/java/pl/allegro/tech/hermes/consumers/consumer/rate/maxrate/HierarchicalCacheMaxRateRegistryTest.java:35]: why this property is not in the `application-intergation.yml`?

Comment (Diewa) [reply hermes-consumers/src/test/java/pl/allegro/tech/hermes/consumers/consumer/rate/maxrate/HierarchicalCacheMaxRateRegistryTest.java:35]: now, properties aren't loaded from the application-integration.yml file. 😐 If I'm not wrong we have to rewrite some part of tests to use spring context (application.yml files). I don't want to do this in this PR

File: hermes-consumers/src/main/resources/application-integration.yml [hunk 2]
```diff
@@ -4,4 +4,76 @@ consumer:
...
+
+kafka:
+  authorization:
+    enabled: false
+    mechanism: PLAIN
+    protocol: SASL_PLAINTEXT
+    username: username
+    password: password
+
+google:
```
Comment (piotrrzysko) [inline hermes-consumers/src/main/resources/application-integration.yml:71]: In my opinion this section should be a child of the `consumer` section. It applies neither to `hermes-frontend` nor `hermes-management`.

File: hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/config/KafkaProperties.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import pl.allegro.tech.hermes.consumers.consumer.receiver.kafka.KafkaParameters;
+
+@ConfigurationProperties(prefix = "kafka")
+public class KafkaProperties {
+
+    private KafkaAuthorizationProperties authorization;
+    private String clusterName = "primary-dc";
+    private String brokerList = "kafka.broker.list, localhost:9092";
```
Comment (piotrrzysko) [inline hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/config/KafkaProperties.java:11]:
```suggestion
    private String brokerList = "localhost:9092";
```
?

File: hermes-consumers/src/main/resources/application-integration.yml (hunk 2 above)
Comment (piotrrzysko) [reply hermes-consumers/src/main/resources/application-integration.yml:71]: OK, I see that this is not directly related to your PR. I will write a comment in #1508.

File: hermes-consumers/src/test/java/pl/allegro/tech/hermes/consumers/consumer/rate/maxrate/MaxRateRegistryTest.java [hunk 4]
```diff
@@ -40,12 +40,12 @@ public class MaxRateRegistryTest extends ZookeeperBaseTest {
     private final ZookeeperPaths zookeeperPaths = new ZookeeperPaths("/hermes");
     private final ConfigFactory configFactory = new MutableConfigFactory();
     private final String consumerId = configFactory.getStringProperty(Configs.CONSUMER_WORKLOAD_NODE_ID);
-    private final String cluster = configFactory.getStringProperty(Configs.KAFKA_CLUSTER_NAME);
+    private final String cluster = "primary-dc";
```
Comment (szczygiel-m) [inline hermes-consumers/src/test/java/pl/allegro/tech/hermes/consumers/consumer/rate/maxrate/MaxRateRegistryTest.java:43]: You can do something like `new KafkaProperties().getClusterName()`, thanks to this we wouldn't have to change this static value anymore.

File: hermes-consumers/src/test/java/pl/allegro/tech/hermes/consumers/supervisor/workload/ConsumerTestRuntimeEnvironment.java [hunk 5]
```diff
@@ -70,6 +73,8 @@ class ConsumerTestRuntimeEnvironment {
 
     private static final int DEATH_OF_CONSUMER_AFTER_SECONDS = 300;
     private final static String CLUSTER_NAME = "primary-dc";
```
Comment (szczygiel-m) [inline hermes-consumers/src/test/java/pl/allegro/tech/hermes/consumers/supervisor/workload/ConsumerTestRuntimeEnvironment.java:75]: shouldn't this be deleted and replaced with value from `kafkaProperties`?

File: hermes-consumers/src/test/java/pl/allegro/tech/hermes/consumers/supervisor/workload/WorkloadRegistryTest.java [hunk 6]
```diff
@@ -31,6 +31,7 @@ public class WorkloadRegistryTest extends ZookeeperBaseTest {
 
     private static final String consumer1 = "consumer1";
     private static final String consumer2 = "consumer2";
+    private static final String clusterName = "primary-dc";
```
Comment (szczygiel-m) [inline hermes-consumers/src/test/java/pl/allegro/tech/hermes/consumers/supervisor/workload/WorkloadRegistryTest.java:34]: same as above, what do You think about `new KafkaProperties().getClusterName()`?

File: integration/src/integration/java/pl/allegro/tech/hermes/integration/AbstractFrontendShutdownTest.java [hunk 7]
```diff
@@ -8,6 +8,10 @@
 import pl.allegro.tech.hermes.test.helper.endpoint.HermesPublisher;
 import pl.allegro.tech.hermes.test.helper.util.Ports;
 
+import static pl.allegro.tech.hermes.common.config.Configs.KAFKA_AUTHORIZATION_ENABLED;
```
Comment (szczygiel-m) [inline integration/src/integration/java/pl/allegro/tech/hermes/integration/AbstractFrontendShutdownTest.java:11]: I think that this static should be from `pl.allegro.tech.hermes.integration.ConfigurationProperties`

**Review by szczygiel-m — APPROVED** [review]

**Pull request merged**